src/main/kotlin/unitfm/UnitApp.kt

```diff
@@ -61,6 +64,19 @@ fun Application.module() {
         return content.toString()
     }
 
+    // Database setup
```

Comment:
Should we start we some rudimentary package/class structure? E.g. move this to `db.Storage` or similar?

Reply:
I think so. Should we use `fm.unit.models`?